node/tests/spaced_utils.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::{anyhow, Result};
+use assert_cmd::cargo::CommandCargoExt;
+use bitcoind::{tempfile::tempdir, BitcoinD, Conf};
+use log::{debug, error};
+use std::{
+    net::{Ipv4Addr, TcpListener},
+    process::{Child, Command},
+    thread,
+    time::Duration,
+};
+
+#[derive(Debug)]
+pub struct SpaceD {
+    pub bitcoind: BitcoinD,
+    process: Child,
+    rpc_port: u16,
+}
+
+const LOCAL_IP: Ipv4Addr = Ipv4Addr::new(127, 0, 0, 1);
+
+impl SpaceD {
+    pub fn new() -> Result<Self> {
+        let mut conf: Conf = Conf::default();
+        // The RPC auth uses username "user" and password "password". If we
+        // don't set this, bitcoind's RPC API becomes inaccessible to spaced due
+        // to auth issues.
+        conf.args = vec!["-regtest", "-fallbackfee=0.0001", "-rpcauth=user:70dbb4f60ccc95e154da97a43b7a9d06$00c10a3849edf2f10173e80d0bdadbde793ad9a80e6e6f9f71f978fb5c797343"];
```

Comment:
Shouldn't this also specify a temp `datadir` for bitcoind?

Reply:
It's part of the default configuration to get a temp directory so we don't need to